install_resources.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I thought that in the mid-future we'll have quite a few resources and that the lambda users might not want to install them all. This script is only a convenience for the tests.

But maybe you have a different perspective on the matter?